dali/pipeline/operators/python_function/util/copy_with_stride_test.cc

```diff
@@ -44,4 +44,49 @@ TEST(CopyWithStrideTest, SimpleCopy) {
   ASSERT_TRUE((out == std::array<uint8, 8>{1, 2, 3, 4, 5, 6, 7, 8}));
 }
 
+TEST(CopyWithStrideTest, OneDimGPU) {
+  float h_data[] = {1, 2, 3, 4, 5, 6};
```

Comment:
Maybe you can reformat to:
```
float h_data[] = {1, 2,
                             3, 4,
                             5, 6};
ASSERT_TRUE((h_out == std::array<float, 3>{1,
                                                                                    3,
                                                                                    5}));
```
So it is more visible what has happened.

Reply:
Such layout would suggest it's two-dimensional data and it's not the case. Although I've restructured the arrays in multidimensional tests.